conf/app.routes

```diff
@@ -110,6 +110,6 @@ GET  /register-your-trust                 uk.gov.hmrc.cgtpropertydisposalsfronte
 
 
 ### Return journey
-GET  /return/start                        uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.ReturnsController.start()
+GET  /return/start                        uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.HomeController.start()
```

Comment:
to avoid confusion we shoujld probably get rid of the references to returns in the comment and the url